src/cdk/a11y/aria-describer/aria-describer.ts

```diff
@@ -218,10 +218,21 @@ export class AriaDescriber implements OnDestroy {
 
   /** Determines whether a message can be described on a particular element. */
   private _canBeDescribed(element: Element, message: string): boolean {
-    return element.nodeType === this._document.ELEMENT_NODE && message != null &&
-        !!`${message}`.trim();
+    if (!this._isElementNode(element)) {
+      return false;
+    }
+
+    const trimmedMessage = message == null ? '' : `${message}`.trim();
+
+    // We shouldn't set descriptions if they're exactly the same as the `aria-label` of the element,
+    // because screen readers will end up reading out the same text twice in a row.
+    return trimmedMessage ? element.getAttribute('aria-label') !== trimmedMessage : false;
```

Comment:
Trim the `aria-label` as well?

Probably not a big deal, but it might make sense to also normalize the whitespace for both.